scanomatic/image_analysis/image_basics.py

```diff
@@ -60,10 +61,11 @@ def Quick_Scale_To_im(
 ) -> np.ndarray:
     if im is None:
         try:
-            im = load_image_to_numpy(path, dtype=np.uint8)
+            im = load_image_to_numpy(str(path), dtype=np.uint8)
         except Exception:
-            _logger.error("Could not open source")
-            return -1
```

Comment:
This -1 didn't seem to be handled anywhere

Reply:
Agree